core/src/test/kotlin/ch/ergon/dope/buildTest/RangeTransformationTest.kt

```diff
@@ -0,0 +1,108 @@
+package ch.ergon.dope.buildTest
+
+import ch.ergon.dope.DopeQueryManager
+import ch.ergon.dope.QueryBuilder
+import ch.ergon.dope.helper.ManagerDependentTest
+import ch.ergon.dope.helper.someNumberArrayField
+import ch.ergon.dope.helper.someStringArrayField
+import ch.ergon.dope.resolvable.expression.unaliased.type.access.get
+import ch.ergon.dope.resolvable.expression.unaliased.type.arithmetic.mul
+import ch.ergon.dope.resolvable.expression.unaliased.type.arithmetic.sub
+import ch.ergon.dope.resolvable.expression.unaliased.type.collection.filter
+import ch.ergon.dope.resolvable.expression.unaliased.type.collection.mapIndexed
+import ch.ergon.dope.resolvable.expression.unaliased.type.function.stringfunction.concat
+import ch.ergon.dope.resolvable.expression.unaliased.type.function.typefunction.toNumber
+import ch.ergon.dope.resolvable.expression.unaliased.type.function.typefunction.toStr
+import ch.ergon.dope.resolvable.expression.unaliased.type.getNumber
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isEqualTo
+import ch.ergon.dope.resolvable.expression.unaliased.type.relational.isLessOrEqualThan
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class RangeTransformationTest : ManagerDependentTest {
```

Comment:
Add a test where we test something like this:
```
SELECT ARRAY {"number": `it`.`no`, "id": `it`.`id`} FOR `it` IN `objectList` END
```

Reply:
done